src/tests/example.spec.js

```diff
@@ -1,27 +1,31 @@
 // @ts-check
+import { getUserCredentials } from '../HelperFunctions.util';
+import 'dotenv/config';
```

Comment:
is this import used in this module?

Reply:
yes, because I use methods from HelperFunctions.util, and parameterizing with .env files